src/tz390.java

```diff
@@ -1824,6 +1810,10 @@ public  class  tz390 {
         {"0D=BASR,RR,20",        //    320 "0D"    "BASR"     "RR"    2 // RPI 1209N
          "4D=BAS,5,50",          //   1150 "4D"    "BAS"      "RX"    5
          "AE=SIGP,10,100",       //   2540 "AE"    "SIGP"     "RS"   10
+         "9C02=RIO,7,70",        //        "9C02"  "RIO"      "S"     7 // #533
+         "9F01=CLRCH,7,70",      //        "9F01"  "CLRCH"    "S"     7 // #533
```

Comment:
Should these 2 lines be above the preceding line???

Reply:
yes, that would be better. Thanks!